### PR TITLE
Draft: App v0.5.4

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

preparing for releasing `0.5.4`

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by manskx — APPROVED** [review]

**Comment by Borda** [comment]
waiting on #13926

File: src/lightning_app/cli/component-template/.github/workflows/ci-testing.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macOS-10.15, windows-2019]
-        python-version: [3.8]
+        os: [ubuntu-20.04, macOS-11, windows-2019]
+        python-version: [3.7]
```
Comment (manskx) [inline src/lightning_app/cli/component-template/.github/workflows/ci-testing.yml:19]:
```suggestion
        python-version: [3.8]
```

Comment (Borda) [reply src/lightning_app/cli/component-template/.github/workflows/ci-testing.yml:19]: Is this critical? I may keep it rather aligned with future/master branch...

Comment (manskx) [reply src/lightning_app/cli/component-template/.github/workflows/ci-testing.yml:19]: not critical

**Review by manskx — APPROVED** [review]

**Pull request merged**